src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -174,6 +176,13 @@ def get_current_batch_groups_from_postgres(project, last_processed_group_id, bat
             "backfill_seer_grouping_records.no_more_groups",
             extra={"project_id": project.id},
         )
+        if enable_ingestion:
+            logger.info(
+                "backfill_seer_grouping_records.enable_ingestion",
+                extra={"project_id": project.id},
+            )
+            project.update_option("sentry:similarity_backfill_completed", int(time.time()))
+
```

Comment:
Shouldn't we wait to set this option until we know the backfill hasn't errored out?

Reply:
once we reach this point, we've completed backfilling for that project. we could error out later, but that project has been completed

Reply:
Ah, I see. I missed the `if total_groups_to_backfill_length == 0` which was hidden in the fold.